### PR TITLE
Fogl 1475: remove the use/requirement of the psycopg2 library in the Python code

### PR DESCRIPTION
**Pull request by sstefano69**
tested in : 
* vbox/Ubuntu server - Development
* raspberry - Deploy

### CODE REVIEW

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 1]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
         },
         "timeout": {
-            "description": "Timeout in seconds for execution of external commands.",
+            "description": "Timeout in seconds for the execution of the external commands.",
             "type": "integer",
             "default": "1200"
         },
         "restart-max-retries": {
-            "description": "Maximum number of retries at restarting Foglamp",
+            "description": "Maximum number of retries at the restart of Foglamp to ensure it is started.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 2]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
         },
         "max_retry": {
-            "description": "Number of retries",
+            "description": "Number of retries for the operations.",
             "type": "integer",
             "default": "5"
         },
         "timeout": {
-            "description": "Timeout in seconds for execution of external commands.",
+            "description": "Timeout in seconds for the execution of the external commands.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 3]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
         },
         "retention": {
-            "description": "Number of backups to maintain. Old backups will be deleted.",
+            "description": "Number of backups to maintain, old ones will be deleted.",
             "type": "integer",
             "default": "5"
         },
         "max_retry": {
-            "description": "Number of retries",
+            "description": "Number of retries for the operations.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 4]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
-            "description": "Directory for semaphores for backup/restore synchronization."
-                           "if not specified, backup-dir will be used.",
+            "description": "Directory used to store semaphores for backup/restore synchronization."
+                           "it uses backup-dir if none.",
             "type": "string",
             "default": "none"
         },
         "retention": {
-            "description": "Number of backups to maintain. Old backups will be deleted.",
+            "description": "Number of backups to maintain, old ones will be deleted.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 5]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
             "description": "Directory where backups will be created, "
-                           "if not specified, FOGLAMP_BACKUP, FOGLAMP_DATA or FOGLAMP_BACKUP will be used.",
+                           "it uses FOGLAMP_BACKUP or FOGLAMP_DATA or FOGLAMP_BACKUP if none.",
             "type": "string",
             "default": "none"
         },
         "semaphores-dir": {
-            "description": "Directory for semaphores for backup/restore synchronization."
-                           "if not specified, backup-dir will be used.",
+            "description": "Directory used to store semaphores for backup/restore synchronization."
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 6]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
         "schema": {
-            "description": "Schema",
+            "description": "Schema for backup and restore operations.",
             "type": "string",
             "default": "foglamp"
         },
         "backup-dir": {
             "description": "Directory where backups will be created, "
-                           "if not specified, FOGLAMP_BACKUP, FOGLAMP_DATA or FOGLAMP_BACKUP will be used.",
+                           "it uses FOGLAMP_BACKUP or FOGLAMP_DATA or FOGLAMP_BACKUP if none.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 7]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
         },
         "database": {
-            "description": "Database to backup/restore",
+            "description": "Database to manage for backup and restore operations.",
             "type": "string",
             "default": "foglamp"
         },
         "schema": {
-            "description": "Schema",
+            "description": "Schema for backup and restore operations.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 8]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
         },
         "port": {
-            "description": "PostgreSQL port",
+            "description": "PostgreSQL port for backup and restore operations.",
             "type": "integer",
             "default": "5432"
         },
         "database": {
-            "description": "Database to backup/restore",
+            "description": "Database to manage for backup and restore operations.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 9]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
...
     _CONFIG_DEFAULT = {
         "host": {
-            "description": "Host server",
+            "description": "Host server for backup and restore operations.",
             "type": "string",
             "default": "localhost"
         },
         "port": {
-            "description": "PostgreSQL port",
+            "description": "PostgreSQL port for backup and restore operations.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 10]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
 
     # Configuration retrieved from the Configuration Manager
     _CONFIG_CATEGORY_NAME = 'BACK_REST'
-    _CONFIG_CATEGORY_DESCRIPTION = 'Backup and Restore'
+    _CONFIG_CATEGORY_DESCRIPTION = 'Configuration for backup and restore operations'
 
     _CONFIG_DEFAULT = {
         "host": {
-            "description": "Host server",
+            "description": "Host server for backup and restore operations.",
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 11]
```diff
@@ -289,58 +289,58 @@ class BackupRestoreLib(object):
 
     # Configuration retrieved from the Configuration Manager
     _CONFIG_CATEGORY_NAME = 'BACK_REST'
-    _CONFIG_CATEGORY_DESCRIPTION = 'Backup and Restore'
+    _CONFIG_CATEGORY_DESCRIPTION = 'Configuration for backup and restore operations'
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: Please revert description

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py (hunk 1 above)
Comment (sstefano69) [reply python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: applied

File: python/foglamp/plugins/storage/postgres/backup_restore/lib.py [hunk 12]
```diff
@@ -308,13 +308,14 @@ class BackupRestoreLib(object):
             "default": "foglamp"
         },
         "schema": {
-            "description": "Schema",
+            "description": "Schema.",
```
Comment (praveen-garg) [inline python/foglamp/plugins/storage/postgres/backup_restore/lib.py]: . ?

**Pull request merged**